bin/varnishd/cache/cache_fetch.c

```diff
@@ -716,7 +716,12 @@ vbf_stp_condfetch(struct worker *wrk, struct busyobj *bo)
 	CHECK_OBJ_NOTNULL(wrk, WORKER_MAGIC);
 	CHECK_OBJ_NOTNULL(bo, BUSYOBJ_MAGIC);
 
-	AZ(vbf_beresp2obj(bo));
+	if (vbf_beresp2obj(bo)) {
+		(void)VFP_Error(bo->vfc, "Could not get storage in vbf_stp_condfetch");
+		vbf_cleanup(bo);
```

Comment:
I think `vbf_cleanup()` is missing?

Reply:
I think it just needs to mirror the failure in a normal fetch:

https://github.com/varnishcache/varnish-cache/blob/29a1a8243dbef3d973aec28dc90403188c1dc8e7/bin/varnishd/cache/cache_fetch.c#L604-L610

Reply:
@rezan I asked myself the same questions, and the answer is here: https://github.com/varnishcache/varnish-cache/pull/3274#issuecomment-608563773

Reply:
Yes @nigoroll is correct, I had fixed this but guess I reverted something and didn't fix it twice.

Reply:
I did the load test with the code from my previous comment and everything worked as described.